packages/lab/src/splitter/SplitHandle.css

```diff
@@ -0,0 +1,198 @@
+.saltSplitHandle {
+  --splitHandle-size: var(--salt-size-thickness-400, calc(var(--salt-size-border) * 4));
+  --splitHandle-borderWidth: var(--salt-size-thickness-100, var(--salt-size-border));
+  --splitHandle-borderColor: var(--salt-separable-secondary-borderColor);
+  --splitHandle-dot-background: var(--salt-separable-foreground);
+  --splitHandle-dot-background-hover: var(--salt-separable-foreground-hover);
+  --splitHandle-dot-background-active: var(--salt-separable-foreground-active);
+
+  position: relative;
+
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2px;
+
+  box-sizing: content-box;
+
+  background: var(--splitHandle-background);
+}
+
+.saltSplitHandle-bordered,
+.saltSplitHandle-variant-primary {
+  --splitHandle-background: var(--salt-container-primary-background);
+  --splitHandle-background-hover: var(--salt-separable-background-hover);
+  --splitHandle-background-active: var(--salt-separable-background-active);
+}
+
+.saltSplitHandle-variant-secondary {
+  --splitHandle-background: var(--salt-container-secondary-background);
+  --splitHandle-background-hover: var(--salt-separable-background-hover);
+  --splitHandle-background-active: var(--salt-separable-background-active);
+}
+
+.saltSplitHandle-variant-tertiary {
+  --splitHandle-background: var(--salt-container-tertiary-background);
+  --splitHandle-background-hover: var(--salt-separable-background-hover);
+  --splitHandle-background-active: var(--salt-separable-background-active);
+}
+
+.saltSplitHandle-variant-transparent {
+  --splitHandle-background: var(--salt-separable-background);
+  --splitHandle-background-hover: var(--salt-separable-background-hover);
+  --splitHandle-background-active: var(--salt-separable-background-active);
+}
+
+.saltSplitHandle[data-resize-handle-state="hover"] {
+  background: var(--splitHandle-background-hover);
+}
+
+.saltSplitHandle[data-resize-handle-state="drag"] {
+  --splitHandle-background: var(--splitHandle-background-active);
+}
+
+.saltSplitHandle[data-panel-group-direction="horizontal"] {
+  flex-direction: column;
+  width: 4px;
```

Comment:
Should this use the split handle size token?

Reply:
Done!